test/karma/constants/objectHelperSpec.js

```diff
@@ -0,0 +1,73 @@
+describe('ObjectHelper', function() {
+  beforeEach(() => window.angular.mock.module('bicker_router'));
+
+  it('gets out what is put in', function() {
+    let a = {};
+    return inject(function(ObjectHelper) {
+      let object = {};
+      ObjectHelper.set(object, 'key', a);
+      return expect(ObjectHelper.get(object, 'key')).toBe(a);
+    });
+  });
+
+  it('allows dot notation to access nested values', function() {
+    let b1 = { c: 22 };
+    let b2 = {};
+
+    return inject(function(ObjectHelper) {
+      let object = {};
+      ObjectHelper.set(object, 'a.b.b1', b1);
+      ObjectHelper.set(object, 'a.b.b2', b2);
+
+      expect(ObjectHelper.get(object, 'a.b')).toEqual(jasmine.objectContaining({b1, b2}));
+      return expect(ObjectHelper.get(object, 'a.b.b1.c')).toEqual(b1.c);
+    });
+  });
+
+  describe('notIn', () =>
+    it('recursively compares two objects, returning an array of dot notation strings representing the paths in a not in b', function() {
+      let a = { user: { id: 1234, address: { suburb: 'Royston' }}};
+      let b = { user: { address: { }}};
+
+      return inject(ObjectHelper => expect(ObjectHelper.notIn(a, b)).toEqual(['user.id', 'user.address.suburb' ]));
+  })
+);
+
+  describe('unset', () =>
+    it('returns true if the field exists and was removed', () =>
+      inject(function(ObjectHelper) {
+        let object = {};
+        ObjectHelper.set(object, 'a.b.c', true);
+        expect(ObjectHelper.unset(object, 'a.b.c.d')).toBe(false);
+        expect(ObjectHelper.unset(object, 'a.b.c')).toBe(true);
+        return expect(object.a.b.c).toBeUndefined();
+      })
+    )
+  );
+
```

Comment:
Get rid of all of these unnecessary returns. Also, replace the arrow functions given to the "it" blocks with function () and replace the function() given to the inject method with arrow functions... again, please apply everywhere, I won't comment on all of them